js/main.js

```diff
@@ -42,8 +44,8 @@ function showFirstNFT() {
     data.nftVisible = null;
     data.nftVisible = (nftData);
 
-    if (data.ratingsInfo[collectionData.collectionName] === undefined) {
-      getCollectionPhotoURL(randomInt);
+    if (!data.ratingsInfo[collectionData.collectionName]) {
+      throttle(getCollectionPhotoURL(randomInt), 3000);
       data.ratingsInfo[collectionData.collectionName] = collectionData;
```

Comment:
This `throttle()` isn't actually doing anything, because `getCollectionPhotoURL` is already being called. If your app is working as intended, then you can simply remove it.